instagram/src/components/UserPage/UserPostings.jsx

```diff
@@ -0,0 +1,30 @@
+import { Link } from "react-router-dom";
+import {
+  PostingBox,
+  Posts,
+  PostImg,
+} from "../../styles/UserPage/UserPostings_style";
+
+function UserPostings() {
+  return (
+    <PostingBox>
+      <Posts>
+        <Link to="/comments">
+          <PostImg />
+        </Link>
+      </Posts>
+      <Posts>
+        <Link to="comments">
+          <PostImg />
+        </Link>
+      </Posts>
+      <Posts>
+        <Link to="comments">
+          <PostImg />
+        </Link>
+      </Posts>
+    </PostingBox>
+  );
+}
```

Comment:
```suggestion
function UserPostings() {
  const paths = [
    { id: 1, to: 'comments'},
    { id: 2, to: 'test'},
    { id: 3, to: 'test2'},
  ];
  return (
  <PostingBox>
    {paths.map((item) => (
        <Posts key={item.id}>
          <Link to={item.to}>
            <PostImg />
          </Link>
        </Posts>
    )}
  </PostingBox>
})
```
지금 비슷한 코드가 반복되고 있는데 새로운 wrapping Data를 만들어서 map을 통해서 반복적인 코드를 줄여주는게 좋음

Reply:
wow

Reply:
👍👍👍👍